src/Microsoft.TemplateEngine.Abstractions/Installer/CheckUpdateResult.cs

```diff
@@ -35,6 +41,11 @@ private CheckUpdateResult(InstallerErrorCode error, string errorMessage, IManage
         /// <remarks>In some cases the version installed can be higher then identified during update check. Since only installer can correctly compare the versions, the installer returns the result if the version is latest rather than letting the caller determine it using string comparison.</remarks>
         public bool IsLatestVersion { get; private set; }
 
+        /// <summary>
+        /// Gets vulnerabilities from checked package.
+        /// </summary>
+        public IReadOnlyList<VulnerabilityInfo>? Vulnerabilities { get; private set; }
```

Comment:
Consider making it non-nullable, unless there is case when nullability is needed.